temporalio/api/cloud/cloudservice/v1/__init__.py

```diff
@@ -194,9 +194,9 @@
 
 # gRPC is optional
 try:
-    import grpc
+    import grpc  # noqa # type: ignore
```

Comment:
Just for my own curiosity, what was failing here that is requiring these two comments?

Reply:
```
temporalio/api/cloud/cloudservice/v1/__init__.py:197:12: F401 `grpc` imported but unused; consider using `importlib.util.find_spec` to test for availability
```

I've added the error code.

The `type: ignore` was to silence Pylance in VSCode. However, our pyright lint passes, so I've removed the `type: ignore` -- I'll add it back in if and when our lint requires it (and perhaps get my IDE type-checking in sync with our CI type-checking)